src/nlayout.rs

```diff
@@ -86,4 +86,28 @@ impl NLayout {
             .map(|i| [i, self.logic_to_phys[i]])
             .collect()
     }
+
+    /// Get physical bit from logical bit
+    fn get_item_logic(&self, logical_bit: usize) -> usize {
```

Comment:
For me, abbreviations that are different words make code harder to read (since there's a small cognitive effort required to suppress the verbatim meaning). It can already be a burden to track what's physical versus logical.

It also seems like the prefix `get_item` here might not really be giving us much.

How about naming this method `logical_to_physical` or `physical_from_logical`?

Reply:
Sure, I'll rename these

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/7977/commits/9640ed1385c1990437115d23df818726f2656eea